packages/skin/src/routes/_index/component/menu-button/+page.marko

```diff
@@ -1260,6 +1260,656 @@
+                    Sport: (+2)
+                </span>
+                <svg
+                    aria-hidden="true"
+                    class="filter-chip__trailing icon icon--12"
+                    width="12"
+                    height="12"
+                >
+                    <use href="#icon-chevron-down-16"/>
+                </svg>
+            </button>
+            <div class="menu-button__menu">
+                <div class="menu-button__items" role="menu">
+                    <div
+                        class="menu-button__item"
+                        role="menuitemradio"
+                        aria-checked="false"
+                    >
+                        <span class="menu-button__checkbox">
+                            <svg
+                                aria-hidden="true"
+                                class="icon icon--18 icon--unchecked"
+                                height="18"
+                                width="18"
+                            >
+                                <use href="#icon-radio-unchecked-18"/>
+                            </svg>
+                            <svg
+                                aria-hidden="true"
+                                class="icon icon--18 icon--checked"
+                                height="18"
+                                width="18"
+                            >
+                                <use href="#icon-radio-checked-18"/>
+                            </svg>
+                        </span>
+                        <span>Menu Item 1</span>
+                    </div>
+                    <div
+                        class="menu-button__item"
+                        role="menuitemradio"
+                        aria-checked="true"
+                    >
+                        <span class="menu-button__checkbox">
+                            <svg
+                                aria-hidden="true"
+                                class="icon icon--18 icon--unchecked"
+                                height="18"
+                                width="18"
+                            >
+                                <use href="#icon-radio-unchecked-18"/>
+                            </svg>
+                            <svg
+                                aria-hidden="true"
+                                class="icon icon--18 icon--checked"
+                                height="18"
+                                width="18"
+                            >
+                                <use href="#icon-radio-checked-18"/>
+                            </svg>
+                        </span>
+                        <span>Menu Item 2</span>
+                    </div>
+                    <div
+                        class="menu-button__item"
+                        role="menuitemradio"
+                        aria-checked="false"
+                    >
+                        <span class="menu-button__checkbox">
```

Comment:
Do we need this class `menu-button__checkbox` and also `menu__radio|checkbox`? I don't see them being defined in the menu-button file. And actually, do we need this extra element?

Reply:
This was copied from filter menu. I think it was used before as a container for checkbox. Will remove since you are right, its not needed.